packages/typespec-ts/test/modularUnit/scenarios/operations/operations.md

```diff
@@ -755,19 +739,19 @@ export async function _testDeserialize(
 
 export function test(
   context: Client,
-  options: TestOptionalParams = { requestOptions: {} },
+  options: TestOptionalParams = { requestOptions: {} }
 ): PagedAsyncIterableIterator<string> {
   return buildPagedAsyncIterator(
     context,
     () => _testSend(context, options),
     _testDeserialize,
     ["200"],
-    { itemName: "lists" },
+    { itemName: "lists" }
   );
 }
 ```
 
-# should generate paging if no @items defined
+# skip: should generate paging if no @items defined
```

Comment:
<img width="1038" height="109" alt="image" src="https://github.com/user-attachments/assets/3b2abf2f-b68a-4393-afdd-9ca4e598f362" />
remove this case?